qa/vagrant/src/test/resources/packaging/utils/plugins.bash

```diff
@@ -176,6 +169,22 @@ install_and_check_plugin() {
     done
 }
 
+# Install a meta plugin
+# $1 - the plugin name
+# $@ - all remaining arguments are jars that must exist in the plugin's
+#      installation directory
+install_meta_plugin() {
```

Comment:
I don't see where this is used. Am I missing something?

Reply:
This is not used in this pull request but we need it for our meta plugins and I think this is where it should be placed.

Reply:
++